### ISSUE TITLE
Reconfiguring network for a new 1GB network card fails: ERROR: Failed to execute command '....monit monitor collectd...: Cannot create socket to [localhost]:2812 -- Connection refused

### ISSUE THREAD

**Issue by Wikinaut** [openmediavault/openmediavault#313]
I inserted a new 1GB LAN network card and want to setup the OMV to work with this instead of the motherboard 100MB LAN.

Running on the OMV omv-firstaid 1 > Configure network interface lead to
```
Configuring network interface. Please wait...
ERROR: Failed to execute command 'export PATH=/bin/.......; export LANG=C.UTF-8; monit monitor collectd 2>&1' with exit code '1':Cannot create socket to [localhost]:2812 -- Connection refused
```

Network card LEDs are off.
No network.

Pls. advice.

**Comment by Wikinaut** [comment]
It works after disabling the on-board LAN network interface.

**Comment by Wikinaut** [comment]
Problem is persistent, coming back again, even after having disabled the onboad LAN IF.

**Comment by Wikinaut** [comment]
And after having rebooted the OMV server, the Web GUI shows:

![grafik](https://user-images.githubusercontent.com/1151915/53688144-91364e00-3d3f-11e9-9a5f-f5963301eacd.png)

**Comment by Darewa** [comment]
Hi,

had the same problem when exchanging motherboard. I researched a little bit and found #113 describing the same problem. Perhaps this might be helpful for you as well.

Well what I did as far as I remember:
1. I stopped mentioned moniting  and rrdcached services via
`service monit stop
service  rrdcached stop
`
2. When I checked known hosts for entries of IPv6 / IPv4 entries AND DELETED THEM:
`bash: cat /etc/hosts
... 
::1             omv.local
192.168.178.214 omv.local
...
bash: cp /etc/hosts /etc/hosts_old
bash: nano /etc/hosts
`
Then I ran 
`omv-firstaid`
In the menu I again started "configure network interface", IPv4 only, DHCP (I use static IP via router) etc and voila it worked again.
